### PR TITLE
chore: Remove Google Recaptcha, Google Web Fonts, and add statement for schools to privacy declaration

### PR DESCRIPTION
**Pull request by CodingDive**
Stale branch name. We are not removing mailchimp

### CODE REVIEW

**Comment by elbotho** [comment]
I don't think we should remove the mailchimp part. We still use mailchimp and you can still subscribe, we just don't embed the form directly on our page. we now link to it.

File: de/privacy/current.md [hunk 1]
```diff
@@ -109,16 +111,6 @@ Sofern Sie gleichzeitig bei Google eingeloggt sind, ist der Google-Service in de
...
-
-Diese Seite nutzt zur einheitlichen Darstellung von Schriftarten so genannte Web Fonts, die von Google bereitgestellt werden. Beim Aufruf einer Seite lädt Dein Browser die benötigten Web Fonts in Deinen Browsercache, um Texte und Schriftarten korrekt anzuzeigen.
-
-Zu diesem Zweck muss der von Dir verwendete Browser Verbindung zu den Servern von Google aufnehmen. Hierdurch erlangt Google Kenntnis darüber, dass über Deine IP-Adresse unsere Website aufgerufen wurde. Die Nutzung von Google Web Fonts erfolgt im Interesse einer einheitlichen und ansprechenden Darstellung unserer Online-Angebote. Dies stellt ein berechtigtes Interesse im Sinne von Art. 6 Abs. 1 lit. f DSGVO dar.
-
-Wenn Dein Browser Web Fonts nicht unterstützt, wird eine Standardschrift von Deinem Computer genutzt.
-
-Weitere Informationen zu Google Web Fonts findest Du unter [hier](https://developers.google.com/fonts/faq) und in der Datenschutzerklärung von Google, [hier](https://policies.google.com/privacy?hl=de).
-
 #### Google reCAPTCHA
```
Comment (hugotiburtino) [inline de/privacy/current.md:114]: You can also remove this. I don't know from any part at serlo.org where we use google recaptcha

File: de/privacy/current.md [hunk 2]
```diff
@@ -275,7 +272,7 @@ Der Versand unseres Newsletters erfolgt über MailChimp, einer E-Mail Versandpla
 
 Die im Rahmen der Registrierung erfassten Daten, also Deine E-Mail Adresse, Deinen Nach- und Vornamen Daten über Deine Interaktion mit dem Newsletter IP-Adresse, Browsertyp und Betriebssystem
```
Comment (hugotiburtino) [inline de/privacy/current.md:274]:
```suggestion
- Die im Rahmen der Registrierung erfassten Daten, also Deine E-Mail-Adresse. 
- Daten über die Interaktion mit dem Registrierungsformular: E-Mail-Addresse, IP-Adresse, Browsertyp und Betriebssystem, Vor- und Nachnamen (optional).
- Daten über Deine Interaktion mit dem Newsletter: IP-Adresse, Browsertyp und Betriebssystem.
```

File: de/privacy/current.md [hunk 3]
```diff
@@ -275,7 +272,7 @@ Der Versand unseres Newsletters erfolgt über MailChimp, einer E-Mail Versandpla
...
 
-Diese Daten werden in der Regeln auf Servern von Mailchimp in den USA gespeichert.
+Mailchimp verwendet für die Datenübertragung in die USA die [Standardvertragsklauseln (SCC)](https://mailchimp.com/legal/data-processing-addendum/) als rechtliche Grundlage, um den Schutz deiner Daten sicherzustellen. Die Daten werden auf Servern in den USA gespeichert.
```
Comment (hugotiburtino) [inline de/privacy/current.md:275]:
```suggestion
Mailchimp verwendet für die Datenübertragung in die USA die [Standardvertragsklauseln (SCC)](https://mailchimp.com/legal/data-processing-addendum/) als rechtliche Grundlage, um den Schutz deiner Daten sicherzustellen.
```
We have already said the data can go to USA, we don't need to repeat it.

By the way, Mailchimp is not written in PascalCase as we can read in their articles (Mailchimp, not MailChimp). We could change all other places, although it is not important.

File: de/privacy/current.md [hunk 4]
```diff
@@ -109,28 +123,6 @@ Sofern Sie gleichzeitig bei Google eingeloggt sind, ist der Google-Service in de
 
 Weitere Informationen von Google über den Umgang mit Nutzerdaten (Datenschutzerklärung) erhalten Sie auf: [http://www.google.com/intl/de/policies/privacy](http://www.google.com/intl/de/policies/privacy)
```
Comment (hugotiburtino) [inline de/privacy/current.md:125]:
```suggestion
Google Custom Search und weitere Produkte von Google LLC sind nach dem Data Privacy Framework zertifiziert, was ein angemessenes Datenschutzniveau gemäß den Vorgaben der DSGVO sicherstellt.
```

**Review by hugotiburtino — APPROVED** [review]

**Pull request merged**